internal/pkg/aws/ec2/ec2.go

```diff
@@ -437,3 +438,25 @@ func (idx *routeTableIndex) IsPublicSubnet(subnetID string) bool {
 	}
 	return idx.mainTable.HasIGW()
 }
+
+// GetManagedPrefixListId returns the PrefixListId of a prefix list queried by name.
+func (c *EC2) GetManagedPrefixListId(prefixListName string) (*string, error) {
```

Comment:
Maybe 
```suggestion
func (c *EC2) ManagedPrefixListID(prefixListName string) (*string, error) {
```
https://go.dev/doc/effective_go#Getters